stats.py

```diff
@@ -7,11 +9,12 @@ def scrape():
     enqueued_jobs = []
     workers = []
 
-    enqueued_jobs = [
-        dict(queue_name=q.name, size=q.count)
+    # Use chain to flatten the list of lists
+    enqueued_jobs = list(chain.from_iterable(
+        [dict(queue_name=q.name, size=q.count),
+         dict(queue_name=f"{q.name}-failed", size=q.failed_job_registry.count)]
```

Comment:
Does this generate a flattened list of all queues/failed queues? It will be helpful to put comments for an example output.

Reply:
It looks like the answer is positive by looking at the tests below. The comments is still helpful though.

Reply:
I had added a comment above explaining that `chain` is being used to flatten the list, you mean I should expand on that to include an example?

Reply:
I didn't pay attention about the comment above in full, but yeah an example will be also good.

Reply:
I am now conditionally extending `enqueued_jobs` depending on the `rq` version.